ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -54,6 +54,27 @@ then
 	source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+# Even if the pods are ready, we need to wait for the webhook server to be
```

Comment:
I would prefer to move this check into `ci-operator/step-registry/openstack/provision/sriov-operator/openstack-provision-sriov-operator-commands.sh` because this steps is only supposed to take care of `SriovNetworkNodePolicy`.